src/core_modules/capture-core/src/components/QuickSelector/QuickSelector.container.js

```diff
@@ -1,10 +1,31 @@
+// @flow
+
 import { connect } from 'react-redux';
 import QuickSelector from './QuickSelector.component';
+import { setOrgUnitId, storeOrgUnitObject, setProgramId, setCategoryId } from './actions/QuickSelector.actions';
 
 const mapStateToProps = (state: Object) => ({
     selectedProgramId: state.currentSelections.programId,
-    selectedOrgUnit: state.currentSelections.orgUnit,
+    selectedCategories: state.currentSelections.categories,
+    // We currently have orgUnit (Object) and orgUnitId (string). Do we need both?
```

Comment:
Move comment to the code adding the state.currentSelections.orgUnit

Reply:
Done.